### ISSUE TITLE
Add support for an additional -ignore tag

### ISSUE THREAD

**Issue by rjayroso** [rubyAnneB/linkReaper-cli#8]
Assuming we saved a file called ignore-urls.txt (which contains links that LinkReaper-cli should ignore):
```bash
# === ignore-urls.txt file ===
# Ignore All Seneca College URLs
https://senecacollege.ca
# Ignore all Learn@Seneca URLs
https://learn.senecacollege.ca/
```
We could use it like this:
```bash
$ linkReaper readfile index.html -ignore ignore-urls.txt
```
This would produce an output without and Seneca College or Learn@Seneca URLs.

**Comment by rubyAnneB** [comment]
go for it

**Comment by rjayroso** [comment]
When you get the time, can you check my changes and push it to your master? Thanks!

**Comment by rubyAnneB** [comment]
I looked at branch issue-8 from your fork
But I couldn't find anything new other than adding the option annotation for one of the commands. Have you pushed everything?

**Comment by rubyAnneB** [comment]
closed with 27306765546ced268b3d35c877e2409f1f492b69
